### ISSUE TITLE
error during setting up the python environment

### ISSUE THREAD

**Issue by EatGoldFish** [GimpelZhang/GaussianRPG#8]
hello, I got this error by run  :   `mamba install ros-humble-desktop`

warning  libmamba Added empty dependency for problem type SOLVER_RULE_UPDATE
Could not solve for environment specs
The following packages are incompatible
├─ python is installable with the potential options
│  ├─ python 3.8.20 would require
│  │  └─ openssl >=3.0.15,<4.0a0 , which can be installed;
│  ├─ python [1.0.1|1.2|...|3.9.9], which can be installed;
│  ├─ python [3.10.0|3.10.1|...|3.10.9], which can be installed;
│  ├─ python [3.11.0|3.11.1|...|3.11.9], which can be installed;
│  ├─ python [3.8.0|3.8.1] would require
│  │  ├─ libffi >=3.2.1,<3.3.0a0 , which can be installed;
│  │  └─ openssl >=1.1.1a,<1.1.2a , which conflicts with any installable versions previously reported;
│  ├─ python [3.8.10|3.8.11|3.8.8] would require
│  │  ├─ libffi >=3.3,<3.4.0a0 , which can be installed;
│  │  └─ openssl >=1.1.1k,<1.1.2a , which conflicts with any installable versions previously reported;
│  ├─ python [3.8.10|3.8.12] would require
│  │  └─ openssl >=1.1.1l,<1.1.2a , which conflicts with any installable versions previously reported;
│  ├─ python 3.8.12 would require
│  │  └─ openssl >=3.0.0,<4.0a0 , which can be installed;
│  ├─ python 3.8.13 would require
│  │  └─ openssl >=1.1.1n,<1.1.2a , which conflicts with any installable versions previously reported;
│  ├─ python 3.8.13 would require
│  │  └─ openssl >=3.0.2,<4.0a0 , which can be installed;
│  ├─ python 3.8.2 would require
│  │  ├─ libffi [>=3.2.1,<3.3.0a0 |>=3.2.1,<3.3a0 ], which can be installed;
│  │  └─ openssl >=1.1.1e,<1.1.2a , which conflicts with any installable versions previously reported;

**Comment by EatGoldFish** [comment]
as shown in this [[page,,](https://robostack.github.io/GettingStarted.html)](https://robostack.github.io/GettingStarted.html) it seems that I should use python3.11

I tried python3.11 and finish `mamba install ros-humble-desktop` , but `pip install setuptools==58.2.0` got another error

Collecting setuptools==58.2.0
  Downloading setuptools-58.2.0-py3-none-any.whl.metadata (4.9 kB)
Downloading setuptools-58.2.0-py3-none-any.whl (946 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 946.1/946.1 kB 193.0 kB/s eta 0:00:00
Installing collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 75.1.0
    Uninstalling setuptools-75.1.0:
      Successfully uninstalled setuptools-75.1.0
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
colcon-common-extensions 0.3.0 requires colcon-notification, which is not installed.

**Comment by EatGoldFish** [comment]
Do you have any ideas? Thanks a lot

**Comment by zzz5y** [comment]
I have the same issue with you.
I think i maybe the system problem
what's the version of your ubuntu?My system is 20.04.

**Comment by EatGoldFish** [comment]
> I have the same issue with you. I think i maybe the system problem what's the version of your ubuntu?My system is 20.04.

ubuntu 22.04, Have you fix it on you system?

**Comment by zzz5y** [comment]
> > I have the same issue with you. I think i maybe the system problem what's the version of your ubuntu?My system is 20.04.
> 
> ubuntu 22.04, Have you fix it on you system?

no,because the work's problem,not enough time to try.Do you make any progress?
